example/my_adapter.js

```diff
@@ -59,13 +62,15 @@ class MyAdapter {
      * - clientId {string} client identifier the token belongs to
      * - aud {array of strings} array of audiences the token is intended for
      * - authTime {number} timestamp of the end-user's authentication
-     * - claims {object} requested claims (see claims parameter in OIDC Core 1.0)
+     * - claims {object} accepted claims (see claims parameter in OIDC Core 1.0), rejected claims
+     *     are removed from the original object
```

Comment:
rejected claims aren't removed anymore. instead there's a `rejected` member (Array of Strings)